app/src/main/res/values/styles.xml

```diff
@@ -27,10 +27,19 @@
     <style name="VocableButton" parent="Widget.AppCompat.Button">
         <item name="android:textColor">@color/vocable_button_text_color</item>
         <item name="android:textAllCaps">false</item>
+        <item name="autoSizeMinTextSize">@dimen/auto_size_min_text_size</item>
+        <item name="autoSizeTextType">uniform</item>
+        <item name="autoSizeStepGranularity">2sp</item>
+    </style>
+
+    <style name="ScalableText">
+        <item name="autoSizeMinTextSize">@dimen/auto_size_min_text_size</item>
+        <item name="autoSizeStepGranularity">@dimen/auto_size_step_granularity</item>
+        <item name="autoSizeTextType">uniform</item>
     </style>
 
     <style name="SpeechButton" parent="VocableButton">
-        <item name="android:textSize">@dimen/speech_button_text_size</item>
+        <item name="autoSizeMaxTextSize">@dimen/speech_button_text_size</item>
```

Comment:
Do we have any `TextViews` not covered by any of the styles? We'd need to add `autoSizeMaxTextSize` to them as well.

Reply:
I'm thinking we might only want to attempt to scale certain parts of the app. A lot of the layouts are using wrap content which makes this behave differently since it requires some amount of constraints to define the area it can scale to.